src/climatevision/generator/industry2018/__init__.py

```diff
@@ -11,16 +11,20 @@
 from . import energy_demand, energy_source
 
 # for mineral industry the energy_use_factor still needs to be added to facts
```

Comment:
@JanLK Do you know what this comment is about? Looks like it might already have been in here before your work? Can we just delete it or is this a real todo?

Reply:
I can't find any missing facts and if I used git log correctly, the line was included in the initial commit.